.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "C_Cpp.intelliSenseEngineFallback": "Enabled"
+}
```

Comment:
Is deleting this file okay? It would be good to keep personal config files out of the project.